models/usersModels.js

```diff
@@ -8,7 +8,14 @@ const userSchema = new mongoose.Schema({
   hotel_id: {
     type: mongoose.Schema.Types.ObjectId,
     ref: MODEL_NAMES.HOTELS,
-    required: true,
```

Comment:
Why don't we want a guest to have a hotel id?

Reply:
Guests will have `hotel_id`. 
This is for Super Admins, as they will not have `hotel_id` when their account will be created with `POST` request to `/api/auth/register` endpoint.